src/jobs/dev-promote-prod.yml

```diff
@@ -5,13 +5,6 @@ description: >
 executor: cli/default
 
 parameters:
-  release:
```

Comment:
I found that validating the orb results in an `Arguments referenced without declared parameters: release` error message as `<<parameters.release>>` is still referenced in the last step of the job